x-pack/plugins/cloud/public/plugin.ts

```diff
@@ -63,5 +76,10 @@ export class CloudPlugin implements Plugin<CloudSetup> {
         href: deploymentUrl,
       });
     }
+
+    if (security && this.isCloudEnabled) {
+      const userMenuLinks = createUserMenuLinks(this.config);
+      security.navControlService.setUserMenuLinks(userMenuLinks);
+    }
```

Comment:
I will let @legrego decide on that, but would it make sense to have a `addUserMenuLinks` API instead of `setUserMenuLinks`. It that supposed to be an 'internal' API between `cloud` and `security`, or something more opened where multiple consumers should be able to register links to be added in the user menu?

Reply:
I agree with @pgayvallet -- I'd prefer to have `addUserMenuLinks` instead of `setUserMenuLinks`. While cloud is the only consumer _so far_, I wouldn't be surprised to find other plugins wanting to augment this at some point in the future.

Reply:
I agree that other plugins may add their own links in the future, so the rename makes sense. I've updated the API.